### DISCUSSION TITLE
Linkwarden

### DISCUSSION THREAD

**Discussion by rkcreation** [coollabsio/coolify#2211] [Service Template Requests]
Linkwarden bookmarks manager

https://github.com/linkwarden/linkwarden/blob/main/docker-compose.yml

**Comment by oregapam** [comment]
Hello @rkcreation 

You can find an "almost native" docker-compose solution in my repo.
Link: https://github.com/oregapam/awesome-coolify-service-templates

Have a nice day.

**Comment by LunaVoid** [comment]
Hi! Just tried your compose and it is spot on. I suggest you make a pull request! This should be a service in the panel.

**Reply by oregapam** [reply]
Hi @LunaVoid thank you for the test and the suggesion.
I'll try to make a pull request.

**Comment by Recreabox** [comment]
Hello, just here to say that would be great to have Linkwarden template in Coolify even if docker compose sent by @oregapam works :)

**Comment by mdahamshi** [comment]
I'd like to work on adding Linkwarden as a native one-click service in Coolify.

The project is currently well above the 1,000 GitHub star requirement, and there is already an existing request for a Linkwarden service template here.

I'll base the template on Linkwarden's official Docker Compose configuration, adapt it to Coolify's service-template conventions, and test it locally before opening a PR.
